.github/pull_request_template.md

```diff
@@ -5,10 +5,10 @@ This PR adds... <!-- A brief description of what your PR does -->
 <!-- This checklist is just a guideline.-->
 <!-- If any of the following does not apply to your PR, please leave them unchecked and explain in the PR description -->
 
-- [ ] I've kept this PR as small as possible (~600 lines) by splitting it into PRs with manageable chunks of code
-- [ ] I've requested reviews from 1-2 reviewers
-- [ ] I've added [accessibility IDs](https://www.notion.so/dc5f4ce910f7431c84344ac79344e9f5?v=d487366816834dd4ab8dc12e0b5928f3) to my components
-- [ ] I've added/updated unit/integration tests
-- [ ] I've added/updated [e2e tests](https://www.notion.so/e2e-Documentation-a096d3e0bf75485e85ad692af8371ef3) for at least the happy flow
-- [ ] I've added [Chinese translations for i18n setup](https://www.notion.so/Translations-Dev-Guide-8c1da838982a4f59a386ec96ac6780c8)
+- [x] I've kept this PR as small as possible (~600 lines) by splitting it into PRs with manageable chunks of code
+- [x] I've requested reviews from 1-2 reviewers
+- [x] I've added [accessibility IDs](https://www.notion.so/dc5f4ce910f7431c84344ac79344e9f5?v=d487366816834dd4ab8dc12e0b5928f3) to my components
+- [x] I've added/updated unit/integration tests
+- [x] I've added/updated [e2e tests](https://www.notion.so/e2e-Documentation-a096d3e0bf75485e85ad692af8371ef3) for at least the happy flow
+- [x] I've added [Chinese translations for i18n setup](https://www.notion.so/Translations-Dev-Guide-8c1da838982a4f59a386ec96ac6780c8)
```

Comment:
What's with these changes :O

Reply:
i thought u meant get back the template?

Reply:
done